index.html

```diff
@@ -44,32 +44,98 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
         </a>
       </label>
       <pre class='f5 sans-serif fw5 ma0 pv2 dib overflow-x-auto w-100' id="hr-cid"></pre>
-      <div class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
+      <div class='f5 sans-serif ma0 charcoal-muted ttu f6 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
       <div class='dn' id="cid" ></div>
+      <span>
+        <p class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>
+        </br> string: base (base)(cid-version - multicodec - multihash)
+        </br> binary: cid_version - multicodec - multihash
+        </p>
+      </span>
     </div>
     <div class='pt4'>
       <label class='db'>
         <a class='pb1 w-100 tracked ttu f6 teal-muted hover-aqua link' href="https://github.com/multiformats/multibase">
           Multibase
         </a>
+       </br>
       </label>
+      <p class='f5 sans-serif ma0 charcoal-muted fw4 pt2'>
+       </br>Self-describing data model for expressing raw binary
+       </br>data in a variety of base-encoding formats.
```

Comment:
nit: we use "data model" only in context of https://ipld.io
https://docs.ipfs.tech/concepts/glossary/#multibase is a protocol for encoding stuff:
```suggestion
       </br>Self-describing protocol for expressing raw binary
       </br>data in a variety of base-encoding formats.
```